### ISSUE TITLE
Add initial external (USB/Bluetooth) keyboard support

### ISSUE THREAD

**Issue by PatrykMis** [Irineu333/SpeakTouch#101]

**Comment by Irineu333** [comment]
I'm going to buy a bluetooth keyboard to test.

**Comment by PatrykMis** [comment]
You don't have to. If you have USB-c adapter you can connect it to Android device and it works like bluetooth.

I think this is lowest priority feature at this stage though.

**Comment by Irineu333** [comment]
If you're going to use it, then it's worth adding.

**Comment by PatrykMis** [comment]
Yes in general, but now there are more important features like WebView support, context menu, accessibility actions, proper reading of compose apps, granularity (ability to read node by character, word, line, paragraph) and many many more from which some are even not listed yet. Worth to have it but it is less important than above.
